addons/website_event_sale/models/sale_order.py

```diff
@@ -40,10 +40,20 @@ def _website_product_id_change(self, order_id, product_id, qty=0, **kwargs):
             values['product_id'] = ticket.product_id.id
             values['event_id'] = ticket.event_id.id
             values['event_ticket_id'] = ticket.id
+            discount = 0
             if order.pricelist_id.discount_policy == 'without_discount':
```

Comment:
I'm just not sure what is `event_booth_pending_ids` in which case there could be an issue

Reply:
Well this line seems strange:

`price_unit=sum(booth.booth_category_id.price_reduce for booth in booths)`

Seems like we should use "price" and do the pricelist conversion as well in case the configuration is "without_discount", since it's a bit the same flow than ticket but for booths in that case, with similar fields (price_reduce, ...).

What do you think?

cc @pko-odoo